src/tools/ethLoader/ethLoaderLib/EthMaintainer.cpp

```diff
@@ -334,7 +334,8 @@ bool EthMaintainer::command_program(eOipv4addr_t ipv4, FILE *programFile, eOupro
     eOipv4addr_t rxipv4addr;
     eOipv4port_t rxipv4port;
 
-    ++progdata.mNProgSteps;
+    // commenting out fixes the programming result
+    //++progdata.mNProgSteps;
```

Comment:
Maybe, you could elaborate more on this @davidetome 
Just to be more verbose and help your future self to understand the cause of the problem 😉

Reply:
done!